twitter-eventlistener-plugin/src/main/java/com/twitter/presto/plugin/eventlistener/TwitterScriber.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.twitter.presto.plugin.eventlistener;
+
+import com.twitter.logging.BareFormatter$;
+import com.twitter.logging.Level;
+import com.twitter.logging.QueueingHandler;
+import com.twitter.logging.ScribeHandler;
+
+import org.apache.thrift.TBase;
+import org.apache.thrift.TException;
+import org.apache.thrift.TSerializer;
+
+import java.util.Base64;
+import java.util.logging.LogRecord;
+
+public class TwitterScriber
+{
+  private static final String DASH = "-";
+  private static final int MAX_QUEUE_SIZE = 1000;
+
+  private QueueingHandler queueingHandler;
+
+  // TSerializer is not thread safe
+  private final ThreadLocal<TSerializer> serializer = new ThreadLocal<TSerializer>()
+  {
+    @Override protected TSerializer initialValue()
+    {
+      return new TSerializer();
+    }
+  };
+
+  /**
+  * Serialize a thrift object to bytes, compress, then encode as a base64 string.
+  * Throws TException
+  */
+  private String serializeThriftToString(TBase thriftMessage) throws TException
```

Comment:
typically we put the constructor first before the methods, followed by public methods then private. would you shift these down please.